### DISCUSSION TITLE
Example of how to access container output stream

### DISCUSSION THREAD

**Discussion by Nicshek** [testcontainers/testcontainers-dotnet#432] [Q&A] [answered]
Hi, I'm trying to build WaitStrategy that watching container's logs for some message, as far as I can see `IWaitForContainerOS UntilMessageIsLogged(Stream stream, string message)` takes input stream as an argument. Does anybody have an example of how to access container output stream, if it's possible?

**Comment by HofmeisterAn** [comment, marked as answer]
There are plenty examples in the unit tests like:

https://github.com/HofmeisterAn/dotnet-testcontainers/blob/f25ed8705fa72c9d03eb185c41bd783db0fbb908/tests/DotNet.Testcontainers.Tests/Unit/Containers/Unix/TestcontainersContainerTest.cs#L291

or

https://github.com/HofmeisterAn/dotnet-testcontainers/blob/ef67133a1287a5cd183af9ae21e56fd44089f69e/src/DotNet.Testcontainers/Configurations/Modules/Databases/CouchbaseTestcontainerConfiguration.cs#L63

**Reply by Nicshek** [reply]
Thanks for the examples
